formats/developers/filter-and-filterset.txt

```diff
@@ -10,43 +10,65 @@ other forms of microscopy.
 Supported Objects
 -----------------
 
-In the schema there is support for the following objects:
-
--  Dichroic - the dichroic mirror or dichromatic beamsplitter. This is
-   used for the primary dichroic of the instrument. Other dichroics in
-   the excitation or emission path can be modelled with the Filter
-   object.
--  Filter - this object is used to represent any type of excitation
-   filter and emission/barrier filter. It contains details of the
-   transmittance of the filter. It can also be used to model a dichroic
-   mirror or dichromatic beamsplitter in the excitation or emission
-   path.
--  FilterSet - a FilterSet belongs to an Instrument in an OME-XML file
-   and has alongside it any Filters and Dichroics it refers to. It is
-   designed to represent a collection of filters available on a certain
-   instrument. It can also represent a filter cube or filter block. To
-   represent the set of filters used to collect an image, LightPath is a
-   better choice. FilterSet may contain zero or more excitation Filters,
-   one Dichroic, and zero or more emission Filters. No ordering of
-   Filters is implied. Filters and Dichroics can be reused across
-   multiple Filter sets. It will be normal to have a single Dichroic
-   that is usable with several Filter combinations.
--  FilterSetRef - these are used to attach a specific filter set from an
-   Instrument to a Channel within an Image. They allow the Filter set to
-   be defined once in the instrument in an OME-XML file, and then used
-   multiple times within the images.
--  ManufacturerSpec - the Manufacturer spec is used by both Dichroic and
-   Filter to store the Manufacturer, Model, SerialNumber and LotNumber
-   of the optical element.
--  FilterWheel - this is a value within a Filter used to record which
-   holder the filter is located in. Although it is named after a wheel,
-   it could just as easily be a filter slider or other mechanism. These
-   holders allow any of their containing Filters to be automatically
-   selected by the microscope control software.
--  LightPath - the LightPath object is a child of Channel and is used to
-   represent the collection of Dichroic and Filters used to create the
-   Channel. FilterSet is on an Instrument granularity, LightPath on a
-   Channel of an Image.
+The schema supports the following objects:
+
+.. glossary::
+
+    Dichroic
+
+        The dichroic mirror or dichromatic beamsplitter. This is
+        used for the primary dichroic of the instrument. Other dichroics in
+        the excitation or emission path can be modelled with the Filter
```

Comment:
More to the point, I already corrected it, I'm not changing it back ;)

On Thu, Dec 3, 2015 at 4:22 PM, Mark Carroll <notifications@github.com>
wrote:

> In formats/developers/filter-and-filterset.txt
> <https://github.com/openmicroscopy/ome-documentation/pull/1343#discussion_r46573285>
> :
>
> > -   it could just as easily be a filter slider or other mechanism. These
> > -   holders allow any of their containing Filters to be automatically
> > -   selected by the microscope control software.
> > --  LightPath - the LightPath object is a child of Channel and is used to
> > -   represent the collection of Dichroic and Filters used to create the
> > -   Channel. FilterSet is on an Instrument granularity, LightPath on a
> > -   Channel of an Image.
> > +The schema supports the following objects:
> > +
> > +.. glossary::
> > +
> > +    Dichroic
> > +
> > +        The dichroic mirror or dichromatic beamsplitter. This is
> > +        used for the primary dichroic of the instrument. Other dichroics in
> > +        the excitation or emission path can be modelled with the Filter
>
> It's technically accepted but distractingly weird. For instance, trying
> now a quick google search of site:gov gives 189k results for "modelled"
> and 11700k results for "modeled", so well under 2%.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openmicroscopy/ome-documentation/pull/1343/files#r46573285>
> .
>